WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -822,4 +864,9 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
     [self maybeShowNUX];
 }
 
+- (UIViewController *)initialDetailViewControllerForSplitView:(WPSplitViewController *)splitView
```

Comment:
Let's add a `mark` above this for the `WPSplitViewControllerDetailProvider`, separating from the `NSFetchedResultsControllerDelegate` mark above.